Library/Homebrew/utils/inreplace.rb

```diff
@@ -5,8 +5,6 @@
 
 module Utils
   # Helper functions for replacing text in files in-place.
-  #
-  # @api private
```

Comment:
The module itself is private though, right? Only the `inreplace` method is a public API.

Reply:
Yes, good catch